packages/evo-marko/src/tags/evo-details/index.marko

```diff
@@ -0,0 +1,52 @@
+export interface Input extends Marko.Input<"details"> {
+  summary?: Marko.AttrTag<Marko.Input<"span">>;
+  size?: "regular" | "small";
+  alignment?: "regular" | "center";
+  as?: keyof Marko.NativeTags;
+}
+
+<const/{
+    class: inputClass,
+    open,
+    alignment,
+    size,
+    summary,
+    content,
+    as: inputAs,
+    // ...htmlInput
+} = input/>
+
+<const/htmlInput=(({
+    class: inputClass,
+    open,
+    alignment,
+    size,
+    summary,
+    content,
+    as: inputAs,
+    ...rest
+}) => rest)(input)/>
+
+<details
+    ...htmlInput
+    class=["details", inputClass]
+    open:=input.open
```

Comment:
The controllable attributes (in this case `openChange`) _can_ be spread, so it's actually completely okay to just remove this line and `open` from the destructure and it will still be controllable